include/eve/module/core/regular/hi.hpp

```diff
@@ -48,14 +48,13 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!      template< eve::value T >
-//!      auto hi(T x) noexcept;
+//!      constexpr auto hi(value auto x)                           noexcept;
```

Comment:
no masking ? We do support it normally now

Reply:
the return type does not allow that